src/main/java/firebase/sdk/dashboard/api/ReleaseResource.java

```diff
@@ -121,6 +121,12 @@ public Response enrollSDKinRelease(@PathParam("releaseName") String releaseName,
     }
 
     Release release = RELEASEDAO.getRelease(platform, releaseName);
+    long currentTime = System.currentTimeMillis() / 1000;
+    // Check if the release is old and if so then dont allow enrollment.
+    if (currentTime > release.codeFreezeTime()) {
+      return ResponseHandler.createJsonResponse(Status.FORBIDDEN, null);
```

Comment:
FORBIDDEN is generally used for cases where you disallow requests from specific clients (while allowing same request from other clients) due to abuse detection, etc. I think you can call this a PRECONDITION_FAILED.